build fails with Java 9

Using JDK 9 the build fails with the following message:
```
> Task :archunit:spotbugsMain FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':archunit:spotbugsMain'.
> SpotBugs rule violations were found. See the report at: file:///Users/joerg/Source/java/ArchUnit/archunit/build/reports/spotbugs/main.html
```

The Spotbug report mentions the following warnings:

```

Code | Warning
-- | --
OBL | com.tngtech.archunit.library.freeze.ViolationStoreFactory$TextFileBasedViolationStore$FileSyncedProperties.loadRulesFrom(File) may fail to clean up java.io.InputStream
  |  
OBL | com.tngtech.archunit.library.freeze.ViolationStoreFactory$TextFileBasedViolationStore$FileSyncedProperties.syncFileSystem() may fail to clean up java.io.OutputStream
```

The same code can be build using JDK 11 so it seems to be a JDK 9 related issue. I used AdoptOpenJDK 9.

--- a/build-steps/codequality/spotbugs-excludes.xml
+++ b/build-steps/codequality/spotbugs-excludes.xml
@@ -46,4 +46,26 @@
         <Method name="readProperties"/>
         <Bug pattern="RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE"/>
     </Match>
+    <Match>
+        <!-- False positive on try-with-resources statement with JDK-9 and JDK-10 (https://github.com/TNG/ArchUnit/issues/199):
+             In method com.tngtech.archunit.library.freeze.ViolationStoreFactory$TextFileBasedViolationStore$FileSyncedProperties.loadRulesFrom(File)
+             Reference type java.io.InputStream
+             1 instances of obligation remaining
+             Obligation to clean up resource created at ViolationStoreFactory.java:[line 192] is not discharged
+        -->
+        <Class name="com.tngtech.archunit.library.freeze.ViolationStoreFactory$TextFileBasedViolationStore$FileSyncedProperties"/>
+        <Method name="loadRulesFrom"/>
+        <Bug pattern="OBL_UNSATISFIED_OBLIGATION"/>
+    </Match>
+    <Match>
+        <!-- False positive on try-with-resources statement with JDK-9 and JDK-10 (https://github.com/TNG/ArchUnit/issues/199):
+             In method com.tngtech.archunit.library.freeze.ViolationStoreFactory$TextFileBasedViolationStore$FileSyncedProperties.syncFileSystem()
+             Reference type java.io.OutputStream
+             1 instances of obligation remaining
+             Obligation to clean up resource created at ViolationStoreFactory.java:[line 214] is not discharged
+        -->
+        <Class name="com.tngtech.archunit.library.freeze.ViolationStoreFactory$TextFileBasedViolationStore$FileSyncedProperties"/>
+        <Method name="syncFileSystem"/>
+        <Bug pattern="OBL_UNSATISFIED_OBLIGATION"/>
+    </Match>
 </FindBugsFilter>

